payment-manager/src/main/java/com/qpay/paymentmanager/config/WebClientProperties.java

```diff
@@ -0,0 +1,23 @@
+package com.qpay.paymentmanager.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "webclient.services")
+@Data
+public class WebClientProperties {
```

Comment:
can you do it using record? https://www.baeldung.com/configuration-properties-in-spring-boot#java-16-records